content/authentication/what-do-you-get/systemuser/_index.nb.md

```
@@ -1,310 +1,175 @@
-![Illustration](illustration4.png "Altinn presenterer ønske fra SmartCloud om opprettelse av systembruker")
-
-![Illustration](illustration4b.png "Ved bekreftelse sender Altinn brukeren tilbake til ønsket side i SmartCloud")
-
-Selve flyten vil variere fra system til system.
-
-I noen tilfeller kan man se for seg at systemleverandøren sender en epost til sluttbruker med lenke til bekreftelse, mens man i fremtiden også potensielt vil kunne se slike forspørsler fra Altinn arbeidsflate.
+- Støtte klientforhold til regnskapsfører/revisor fra Q2 2025
+- Støttes av Altinn Apps Q1 2025
 
-### Leverandørstyrt opprettelse av Agent systembruker
+**For sluttbruker**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)